react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -8,15 +8,14 @@ function Board({ squares, onClick }) {
   const [firstRow, secondRow, thirdRow] = boardRows;
 
   return (
-    <fragment>
+    <div>
```

Comment:
You can use <> ... </> instead 

Use divs only if you need to set a style on a wrapper container. Most of the times when you have just a div without classes it means you can delete it.